backend/webserver/serializers.py

```diff
@@ -10,9 +10,16 @@ class Meta:
 
 class PostSerializer(serializers.ModelSerializer):
     author = AuthorSerializer(read_only=True)
+    likesCount = serializers.SerializerMethodField('count_likes')
+    
     class Meta:
         model = Post
-        fields = ['id','author','created_at','edited_at','title','description','source','origin','unlisted','content_type','content','visibility']
+        fields = ['id','author','created_at','edited_at','title','description','source','origin','unlisted','content_type','content','visibility',"likesCount"]
```

Comment:
You just added a field here. Where is the corresponding view change? Just adding a field here doesn't do anything? Where is a test case that tests this new addition? Did you update the documentation with the likes count?

Reply:
I would appreciate it if you could double-check your work before asking me to review it. It really does not help our priorities if I am constantly spotting bugs or missing pieces every time I review your PRs. I really need you to think and figure out these things by yourself because otherwise it is the same as me doing the work.